app/experimenter/reporting/api/v7/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from experimenter.reporting.api.v7.views import ReportDataView
+
+urlpatterns = [
+    url(
+        r"^(?P<start_date>\d{4}-\d{2}-\d{2})/(?P<end_date>\d{4}-\d{2}-\d{2})$",
```

Comment:
You know this is kind of a neat idea.  Usually I'd use query parameters for parameters like this, but putting it in the url lets you enforce the date format.  There's some [discussion](https://stackoverflow.com/questions/30967822/when-do-i-use-path-params-vs-query-params-in-a-restful-api) here related.  Fundamentally it's just about patterns there's no hard and fast rule.  

Maybe according to what they're saying here the reportlogs is the resource and the dates are filters, so they'd go in query parameters.  If you're the only one calling it then you can structure it however you like, but one thing to think about is, if some somewhere starts calling this API, and you need to move something around that would break the calling interface, how flexible is this design?  For instance what if in the future you need to filter on date with time?  Or make either start/end date optional?  Something to think about.

Reply:
so I initially wanted to do query parameters where it would've been like `/v7/?startdate=2020-11-11&enddate=2020-12-12` had some trouble, don't remember why (probs cause I suck at regex), but I think I might take another stab at it as another issue?

Reply:
We all suck at regex's, @tiftran 🙏 😆